mote/templates/aboutpage.html

```diff
@@ -0,0 +1,746 @@
+<!doctype html>
+<!--
+##########################################################################
+*
+*   Copyright © 2019-2021 Akashdeep Dhar <t0xic0der@fedoraproject.org>
+*
+*   This program is free software: you can redistribute it and/or modify
+*   it under the terms of the GNU General Public License as published by
+*   the Free Software Foundation, either version 3 of the License, or
+*   (at your option) any later version.
+*
+*   This program is distributed in the hope that it will be useful,
+*   but WITHOUT ANY WARRANTY; without even the implied warranty of
+*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*   GNU General Public License for more details.
+*
+*   You should have received a copy of the GNU General Public License
+*   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*
+##########################################################################
+-->
+<html lang="en">
+    <head>
+        <meta charset="utf-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1"/>
+        <link href="{{ url_for('static', filename='css3/bootstrap.min.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/custom_fonts.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/fragment.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/fontawesome-all.min.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/fullcalendar.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/about.css') }}" rel="stylesheet"/>
+        <script src="{{ url_for('static', filename='jscn/bootstrap.bundle.min.js') }}"></script>
+        <script src="{{ url_for('static', filename='jscn/jquery.min.js') }}"></script>
+        <link rel="icon" href="{{ url_for('static', filename='imgs/logoteal.png') }}" sizes="16x16">
+        <script src="{{ url_for('static', filename='jscn/fragment.js') }}"></script>
+        <script src="{{ url_for('static', filename='jscn/easy.qrcode.min.js') }}"></script>
+        <script src="{{ url_for('static', filename='jscn/socket.io.js') }}"></script>
+        <script src="{{ url_for('static', filename='jscn/fullcalendar.js') }}"></script>
+        <title>Meetbot Logs</title>
+    </head>
+    <body class="body bg-light">
+        <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-light" style="background-image: linear-gradient(180deg,#eee 0,#ddd);">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="#" style="width: 175px; margin-right: 0px;">
```

Comment:
Add a link back to the home page here.
Also, the "Recent meetings" menu item should also link back to the home page.